lms/djangoapps/discussion/tasks.py

```diff
@@ -113,8 +113,10 @@ def _is_not_subcomment(comment_id):
 
 def _is_first_comment(comment_id, thread_id):
     thread = cc.Thread.find(id=thread_id).retrieve(with_responses=True)
-    first_comment = thread.children[0]
-    return first_comment.get('id') == comment_id
+    if getattr(thread, 'children', None):
```

Comment:
I think a better conditional is 
``` 
if thread.children and len(thread.children) > 0:
```

Reply:
We have to use `getattr`, since the error is that `thread` doesn't have a `children` attribute.  Probably this will do it in the smallest number of characters:
```python
if getattr(thread, 'children', []):
    # do the stuff
```

Reply:
...although, using `None` as the default, as Rabia already has done, will work fine, too.

Reply:
yes, code will work fine by using `getattr(thread, 'children', None)` in both cases i.e thread doesn't have attribute children or len(thread.children) = 0.